frontend/pages/journey/[id].vue

```diff
@@ -676,21 +753,135 @@ const handleUpload = (result: string) => {
                                     <div
-                                        class="relative flex h-full w-full flex-col items-end"
+                                        class="relative flex h-full w-full flex-col overflow-hidden"
                                     >
-                                        <img
-                                            class="absolute right-[50%] top-[25%] z-20 w-40 -translate-y-[25%] translate-x-[50%] max-sm:mt-1"
-                                            :src="qrcode"
-                                            alt="QR Code"
-                                        />
                                         <div
-                                            class="absolute bottom-4 right-2 z-40 ml-10 flex h-16 w-16 items-center justify-center self-center rounded-full border-2 border-dashed border-natural-400 pl-1.5 pr-1.5 text-xs text-natural-400 dark:border-natural-50 dark:text-natural-50"
+                                            class="absolute bottom-4 right-2 z-40 ml-10 flex h-16 w-16 cursor-pointer select-none items-center justify-center self-center rounded-full border-2 border-dashed border-natural-400 pl-1.5 pr-1.5 text-xs text-natural-400 dark:border-natural-50 dark:text-natural-50"
                                         >
                                             <T key-name="journey.turn" />
                                         </div>
-                                        <SvgStripes
-                                            class="z-0 md:w-2/3 lg:w-1/2"
+                                        <SvgOtherStripes
+                                            class="absolute bottom-1 z-0 -ml-1 w-32 self-start pb-2 md:w-2/3 lg:w-1/2"
                                         />
+                                        <div class="flex h-full">
+                                            <div class="z-0 ml-6 h-full w-1/2">
+                                                <div
+                                                    class="-ml-14 mt-3 flex h-1/2 items-center justify-center"
+                                                >
+                                                    <WeatherIcon
+                                                        class="-mt-2 ml-8 w-24"
+                                                        :weather-code="
+                                                            weatherCodeToday
+                                                        "
+                                                    />
+                                                </div>
+                                                <div class="flex">
+                                                    <div>
+                                                        <div
+                                                            class="mt-1 flex h-full w-full items-center justify-start text-5xl text-text dark:text-natural-50"
```

Comment:
![image](https://github.com/user-attachments/assets/d1ec845a-04df-4619-ab7a-2f9d098725fd)
can u move the description more to the middle so its under the temperature, please